src/i18n/index.js

```diff
@@ -1,12 +1,15 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { IntlProvider, addLocaleData } from 'react-intl';
-import en from 'react-intl/locale-data/en';
-import es from 'react-intl/locale-data/es';
+import { IntlProvider } from 'react-intl';
 import { connect } from 'react-redux';
 import reducer from './reducer';
+import '@formatjs/intl-pluralrules/polyfill';
+import '@formatjs/intl-pluralrules/dist/locale-data/en';
+import '@formatjs/intl-pluralrules/dist/locale-data/es';
+import '@formatjs/intl-relativetimeformat/polyfill';
+import '@formatjs/intl-relativetimeformat/dist/locale-data/en';
+import '@formatjs/intl-relativetimeformat/dist/locale-data/es';
```

Comment:
We need to find a way to import this conditionally since these polyfills are only needed in browsers that do not support Intl.PluralRules and Intl.RelativeTimeFormat.